tests/telemetry/test_series.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from ddtrace.internal.telemetry.data.host import get_hostname
+from ddtrace.internal.telemetry.data.metrics import Series
+
+
+def test_series_to_dict():
+    series = Series("test.metric")
+
+    assert series.to_dict() == {
+        "metric": "test.metric",
+        "points": [],
+        "tags": {},
+        "type": Series.COUNT,
+        "common": False,
+        "interval": None,
+        "host": get_hostname(),
+    }
+
+
+def test_series_add_tag():
+    series = Series("test.metric")
+
+    series.add_tag("foo", "bar")
+    assert series.tags == {"foo": "bar"}
+
+
+def test_series_add_point():
+    series = Series("test.metric")
+
+    series.add_point(111111)
+    series.add_point(222222)
+
+    curr_time_in_secs = int(time.time())
```

Comment:
this is going to be flaky since the time can change between the 2 calls to `add_point` and not, instead you should mock over `time.time` and give a constant value(s)